pipeline.groovy

```diff
@@ -279,4 +279,8 @@ DFLAGS=-I%@P%/../imports -L-L%@P%/../libs -L--export-dynamic -L--export-dynamic
     stage ('Test Projects') {
         parallel mapSteps(dub_projects, this.&testDownstreamProject)
     }
+
+    stage ('Cleanup') {
+        sh 'find $HOME/.dub/packages -type d -name .dub -exec rm -r {} +'
```

Comment:
It doesn't yet use the `dmd --version` reported version (which includes a commit hash).
https://github.com/dlang/dub/blob/038fc1b471656ed76217f7f9da81a361a6f68b10/source/dub/generators/build.d#L344